easybuild/easyconfigs/e/ELPA/ELPA-2016.05.004-intel-2016b.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Authors::   Inge Gutheil <i.gutheil@fz-juelich.de>, Alan O'Cais <a.ocais@fz-juelich.de>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'ELPA'
+version = '2016.05.004'
+
+homepage = 'http://elpa.rzg.mpg.de'
+description = """Eigenvalue SoLvers for Petaflop-Applications ."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://elpa.mpcdf.mpg.de/html/Releases/%(version)s/']
+sources = ['elpa-%(version)s.tar.gz']
+
+builddependencies = [('Autotools', '20150215')]
+
+configopts = '--with-generic-simple --disable-shared '
```

Comment:
This was how the ELPA easyconfig was originally contributed. I can try dropping that, and see what we end up with.